content/python/vlanTagging.md

```diff
@@ -1,18 +1,164 @@
 ---
-title: "Tag Vlans"
-description: "Some examles of how to add vlans to a server. AKA Vlan Trunks."
-date: "2017-06-09"
+title: "Trunking VLANs"
+description: "Some examles of how to adding VLAN Trunks. AKA VLAN Tagging"
+date: "2022-08-04"
 classes: 
     - "SoftLayer_Network_Component"
 tags:
     - "vlan"
     - "server"
 ---
+This example shows how to add VLAN Trunks to a server.
 
-A rough examples of how to add vlans to servers
+*NOTE* When trying to get the vlan tags/trunks from a server, the trunk information is on the `Hardware_Server->networkComponent->uplinkComponent->vlanTrunks` relationship. Esentially the Server's NIC doesn't know about network VLANs, but the switch port that is connected to the NIC does.
```

Comment:
networkComponent->uplinkComponent->networkVlanTrunks* is the path.

I'd probably remove the 2nd sentence starting with "Essentially", which doesn't add much value IMO.